R/utils-bin.R

```diff
@@ -160,6 +167,7 @@ findBinWidth.Date <- function(x, na.rm = c(FALSE, TRUE)) {
 #' @importFrom grDevices nclass.FD
 #' @importFrom grDevices nclass.Sturges
 findNumBins <- function(x) {
+  if (!length(x)) { return(NULL) }
```

Comment:
Does this also return null (or another expected value) if all values in x are NA?

Reply:
trying to keep internally consistent here. binWidth and numBins within plot.data are always either a number or NULL. logic on how to provide binWidth of numBins back to the client is up to the plot class.